src/views/About.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+<div class="background">
+<!-- Bootstrap container für den Abstand und die mittige Platzierung !-->
+<b-container center>
+    <div>
+
+        <b-row>
+            <b-col>
+                <!-- Überschrift  !-->
+                <h1 class="u">ÜBER UNS</h1>
+                <br><br>
+            </b-col>
+        </b-row>
+
+        <b-row>
+            <b-col class="px-0">
+                <!-- Gruppenbild !-->
+                <b-img class="image" src="https://picsum.photos/1024/400/?image=41" rounded fluid-grow alt="Responsive image"></b-img>
+            </b-col>
+        </b-row>
+        <br>
+        <!-- Liste von Namen !-->
+        <b-row class="d-flex justify-content-between names">
+
+            <!-- Erste Spalte !-->
+            <b-col cols="12" md="4" class="d-flex justify-content-start flex-column align-items-start">
```

Comment:
Durch die align-items Klasse in den Cols wird auch mobil die Liste komisch geordndet.